src/boundaries.cpp

```diff
@@ -314,7 +314,9 @@ void fields::connect_the_chunks() {
     FOR_H_AND_B(hc, bc) {
       B_redundant[5 * (num_chunks + i) + bc - Bx] = chunks[i]->f[hc][0] == chunks[i]->f[bc][0];
     }
+  am_now_working_on(Other);
```

Comment:
Shouldn't this be MPITime rather than Other?

Reply:
Fixed by 1972531.